jobclient/src/main/java/com/twosigma/cook/jobclient/Job.java

```diff
@@ -93,10 +93,11 @@ public static Status fromString(String name) {
         private Boolean _isMeaCulpaRetriesDisabled;
         private List<Instance> _instances = Collections.synchronizedList(new ArrayList<Instance>());
         private List<FetchableURI> _uris = new ArrayList<>();
-        private Map<String,String> _env = new HashMap<>();
+        private Map<String, String> _env = new HashMap<>();
         private JSONObject _container;
-        private Map<String,String> _labels = new HashMap<>();
+        private Map<String, String> _labels = new HashMap<>();
         private List<UUID> _groups = new ArrayList<>();
+        private List<TaskConstraint.Constraint> _constraints = new ArrayList<>();
```

Comment:
Should this be a set instead of a list?

Reply:
I prefer `List` here. Is there any reason we use `Set`.

Reply:
`Set` will mean that we cannot repeat the same constraint. With lists, it will be possible to repeat a constraint and will unnecessarily be transmitted in the JSON.

Reply:
That make sense. Will change that accordingly